pkg/packetfilter/fake/namedset.go

```diff
@@ -0,0 +1,307 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"github.com/submariner-io/submariner/pkg/packetfilter"
+	"k8s.io/utils/set"
+)
+
+type namedType struct {
```

Comment:
Minor thing but more clear.

```suggestion
type namedSet struct {
```

Reply:
Its just a minor thing, but just want to check if you wanted to use namedSet here? (as the conversation is marked as resolved).

Reply:
> Its just a minor thing, but just want to check if you wanted to use namedSet here? (as the conversation is marked as resolved).

Ooops,   missed it. 
thanks